include/dl/dl_typelib.h

```diff
@@ -19,19 +19,33 @@ extern "C" {
 
 #include <dl/dl.h>
 
+/*
+	Function pointer type: dl_include_handler
+	    Is responsible for locating and loading a type-library into dl_ctx.
+
+	Parameters:
+	    handler_ctx     - context passed in to dl_context_load_txt_type_library_with_includes()
+	    dl_ctx          - dl-context to load typelib into.
+	    file_to_include - the path to the file to include
```

Comment:
Is something like include_identifier a better name as it actually isn't required to be a name?

Reply:
Sure